solr/solr-ref-guide/modules/upgrade-notes/pages/major-changes-in-solr-10.adoc

```diff
@@ -60,6 +60,11 @@ Users who previously relied on collection-specific URLs to avoid including the c
 
 The service installer now installs a `systemd` startup script instead of an `init.d` startup script. It is up to the user to uninstall any existing `init.d` script when upgrading.
 
+=== SolrCloud request routing
+
+HTTP requests to SolrCloud that are for a specific core must be delivered to the node with that core.
+Past SolrCloud versions would scan all cores everywhere to find a node with the core, and if found then proxy the request.
```

Comment:
Feels like a slightly dangling sentence...   I now understand the "past", now what is happening?   Or was this just a a inefficient bug pattern?

Reply:
> now what is happening

Will HTTP 404 if Solr can't resolve the request.  I should add that.  I don't find the last sentence to be dangling but feel free to offer a rewording suggestion.